ui/hooks/src/use-integration-registry.ts

```diff
@@ -112,8 +112,9 @@ const getAllIntegrationsIds = async (offset?: number) => {
 /**
  * Hook to get all the published integrations ids
  */
-export function useGetAllIntegrationsIds(offset?: number) {
+export function useGetAllIntegrationsIds(enabler?: boolean, offset?: number) {
   return useQuery([INTEGRATIONS_KEY, 'ids'], () => getAllIntegrationsIds(offset), {
+    enabled: !!enabler,
```

Comment:
this should be false by default? if i don't pass the enabler (because it's optional) then it will be false. I would keep it default to true.

Reply:
I'm referring to the case when you would call this hook without specifying the `enabler` like this:
`useGetAllIntegrationsIds()`.
In this case, `enabler` is `undefined` so `!!undefined` = false